sky/core.py

```diff
@@ -109,6 +109,35 @@ def status(cluster_names: Optional[Union[str, List[str]]] = None,
                                       cluster_names=cluster_names)
 
 
+@usage_lib.entrypoint
+def service_status(service_name: Optional[str]) -> List[Dict[str, Any]]:
+    return backend_utils.refresh_service_status(service_name)
+
+
+@usage_lib.entrypoint
+def serve_tail_logs(service_record: Dict[str, Any], replica_id: int,
```

Comment:
As an API, people not know how to create/get a `service_record`. Can we instead take `service_name`?